### ISSUE TITLE
BT-10: Main activity (of the buyer)

### ISSUE THREAD

**Issue by ColinMaudry** [open-contracting/european-union-support#64]
This BT is bound the [Main activity code list](https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/main-activity).

This list contains many values that:

- are not from COFOG
- seem mostly related to private sector

@JachymHercher Is it safe to assume that only the COFOG values will be used with BT-10, since it applies to public organizations? That would simplify the guidance as I'm not sure what sort of code scheme the other values come from.

Merci !

**Comment by JachymHercher** [comment]
> Is it safe to assume that only the COFOG values will be used with BT-10, since it applies to public organizations?

Nope. The Main activity code list regrettably mixes what I would consider two conceptually different codelists - COFOG (used for BT-10 Activity Authority) and the list of activities based on Directive 2014/25/EU (used for BT-610 Activity Entity), where each code starting with "activity" corresponds to one of the articles between Art. 8 and Art. 14. Which one is used depends on various complicated rules - whether you are a contracting authority or a contracting entity, what you are buying, etc.

**Comment by ColinMaudry** [comment]
I've used the same guidance for BT-10 and BT-610.
